### PR TITLE
Spartan Edits

### PR DESCRIPTION
**Pull request by Stingray540**
:cl: Stingray
mapedit: Minor edits to the ONI Aegis
tweak: Edits Spartan SD to be easier to see the active death range for it, gives them a UI icon to activate it. Also gives Spartan helmets protection against welding.
/:cl:

### CODE REVIEW

File: code/modules/halo/clothing/spartan_armour.dm [hunk 1]
```diff
@@ -62,6 +65,9 @@
 	item_state_slots = list(slot_l_hand_str = "syndicate-black", slot_r_hand_str = "syndicate-black")
 	var/list/available_abilities = list()
 
+/obj/item/clothing/suit/armor/special/spartan/ui_action_click()
```
Comment (X0-11) [inline code/modules/halo/clothing/spartan_armour.dm]: This interferes with like literally all armour special abilties that require a ui button to activate.

File: code/modules/halo/clothing/special_armour_procs.dm [hunk 2]
```diff
@@ -50,7 +50,7 @@
 						if(m_kill == our_mob)
 							continue
 						m_kill.dust()
-					explosion(our_mob.loc,0,1,2,7,guaranteed_damage = 77,guaranteed_damage_range = 2)
+					explosion(our_mob.loc,1,2,3,7,guaranteed_damage = 77,guaranteed_damage_range = 2)
```
Comment (X0-11) [inline code/modules/halo/clothing/special_armour_procs.dm]: Light damage radius should stop at the guaranteed damage range, due to this being for the sake of easily seeing the self ddestruct radius

**Review by X0-11 — DISMISSED** [review]

File: code/modules/halo/clothing/spartan_armour.dm (hunk 1 above)
Comment (Stingray540) [reply code/modules/halo/clothing/spartan_armour.dm]: Removed per new commit.

File: code/modules/halo/clothing/special_armour_procs.dm (hunk 2 above)
Comment (Stingray540) [reply code/modules/halo/clothing/special_armour_procs.dm]: Done, I think

**Review by X0-11 — APPROVED** [review]

**Pull request merged**